backend/src/files/files.service.ts

```diff
@@ -68,4 +74,63 @@ export class FilesService {
 			throw new NotFoundException();
 		}
 	}
+
+	async exportMarkdown(
+		exportFileRequestBody: ExportFileRequestBody
+	): Promise<ExportFileResponseDto> {
+		const { exportType, content, fileName } = exportFileRequestBody;
+
+		try {
+			switch (exportType) {
+				case "markdown":
+					return this.exportToMarkdown(content, fileName);
+				case "html":
+					return this.exportToHtml(content, fileName);
+				case "pdf":
+					return this.exportToPdf(content, fileName);
+				default:
+					throw new BadRequestException("Invalid export type");
```

Comment:
We can remove this lines by adding `class-validator`.
Please refer other comments.